server/modules/category/repository/categoryRepository.js

```diff
@@ -240,6 +267,88 @@ export const fetchIndex = async (surveyId, draft = false, client = db) =>
     (indexItem) => dbTransformCallback(indexItem, draft, true)
   )
 
+export const fetchCategoryCodesListStream = (surveyId, categoryUuid, levels, headers, languages) => {
```

Comment:
call this getCategoryStream

Reply:
move this method to its own file, and create methods with jsdoc (name is getCategoryStream)

Reply:
Renamed to getCategoryStreamAndHeaders as we need the headers too